great_tables/_formats.py

```diff
@@ -2568,17 +2571,22 @@ def _replace_minus(string: str, minus_mark: str) -> str:
     return _str_replace(string, "-", minus_mark)
 
 
-def _filter_pd_df_to_row(pd_df: pd.DataFrame, column: str, filter_expr: str) -> pd.DataFrame:
-    filtered_pd_df = pd_df[pd_df[column] == filter_expr]
+T_dict = TypeVar("T_dict", bound=TypedDict)
+
+
+# TODO: remove pandas
+def _filter_pd_df_to_row(pd_df: "list[T_dict]", column: str, filter_expr: str) -> T_dict:
```

Comment:
All locale info went from being a DataFrame, to a list of dictionaries (i.e. list of rows). This function just got the row where some column was equal to something.